llvm/lib/Target/AMDGPU/SIInsertWaitcnts.cpp

```diff
@@ -165,6 +165,18 @@ enum VmemType {
   NUM_VMEM_TYPES
 };
 
+static unsigned getRegPoint(const GCNSubtarget &ST, MCRegister Reg,
+                            const SIRegisterInfo &TRI) {
```

Comment:
Since after https://github.com/llvm/llvm-project/pull/130056 we only compute 'register points' in one single place, I think there's no point in having `getRegPoint()` as a separate function anymore.

Reply:
removed getRegpoint